src/odemis/gui/cont/acquisition.py

```diff
@@ -815,9 +817,11 @@ def on_acquisition_done(self, future):
             logging.error("Acquisition failed (after %d streams): %s",
                           len(data), exp)
 
-        # Remove live update
+        #  REMOVE the the overlay (live_update) in the SEM window which displays the SEM measurements of the current
+        #  acquisition if a  stream is added and acquisition is started.
+        # TODO K.K. fix that also the CL stream is removed
```

Comment:
No, I think the CL stream should stay, as it's a setting stream. That said, you might want to make sure the last data displayed corresponds to the complete acquisition (without any orange/blue indicators).... but that not a big deal if it's not done and just left as a TODO.